src/components/Layout/Footer.tsx

```diff
@@ -151,13 +163,7 @@ const Footer: React.FC<React.HTMLProps<HTMLDivElement>> = (props) => {
           </a>
           <a
             className={styles.link}
-            href="https://support.maxmind.com/"
-          >
-            Frequently Asked Questions
-          </a>
```

Comment:
Was removing the FAQ link intentional? I didn't see it in the AC so just wanted to ask.

Reply:
We no longer have the FAQs on support.maxmind.com. This was a change that should have been implemented in a prior branch, but I forgot to do so then.